keanu-project/src/main/java/io/improbable/keanu/vertices/bool/nonprobabilistic/BooleanProxyVertex.java

```diff
@@ -30,11 +31,19 @@ public BooleanProxyVertex(VertexLabel label) {
     }
 
     @ExportVertexToPythonBindings
-    public BooleanProxyVertex(long[] shape, VertexLabel label) {
+    public BooleanProxyVertex(long[] shape,VertexLabel label) {
         super(shape);
         setLabel(label);
     }
 
+    public BooleanProxyVertex(@LoadShape long[] shape, @LoadVertexParam(LABEL_NAME) VertexLabel label, @LoadVertexParam(value = PARENT_NAME, isNullable = true) BooleanVertex parent) {
```

Comment:
Could we stick with just a constructor that takes a string for loading?  The more types the save/load code has to support the sadder I am :-)